source/portable/mbedtls/core_pkcs11_mbedtls.c

```diff
@@ -661,7 +693,7 @@ static CK_RV prvRsaKeyAttParse( const CK_ATTRIBUTE * pxAttribute,
             break;
 
         default:
-            xResult = CKR_ATTRIBUTE_TYPE_INVALID;
+            /* Unreachable */
```

Comment:
Unreachable even when there is a bug somewhere ?
maybe add an assert?

Reply:
Added a comment and set the result